core/src/autogluon/core/models/ensemble/bagged_ensemble_model.py

```diff
@@ -108,6 +114,13 @@ def _fit(self,
              time_limit=None,
              sample_weight=None,
              **kwargs):
+        use_child_oof = self.params.get('use_child_oof', False)
+        if use_child_oof:
```

Comment:
Will BaggedEnsemble with `_child_oof=True` be unnecessarily retrained during refit_full()? Or is this also covered by the `self.is_fit()` check here?

Reply:
It will be unnecessarily retrained. The code to make this not occur is complex, so it can be left to a future PR.

Reply:
Added a FIXME comment regarding this